main.go

```diff
@@ -401,6 +401,8 @@ func startServices(ctx context.Context, majordomo majordomo.Service, monitor met
 			return errors.Wrap(err, "failed to obtain account generation passphrase for process")
 		}
 	}
+
+	viper.SetDefault("process.generation-timeout", 10*time.Second)
```

Comment:
There is a defaults section in the `fetchConfig()` function, this default should go there.